CHANGELOG.md

```diff
@@ -1,3 +1,139 @@
+* [FEATURE] Allow relabeling the protocol scheme of targets.
+* [FEATURE] Add `predict_linear()` query language function.
+* [FEATURE] Support for authentication using bearer tokens, client certs, and
+  CA certs.
+* [FEATURE] Implement unary expressions for vector types (`-foo`, `+foo`).
+* [FEATURE] Add console templates for the SNMP exporter.
+* [FEATURE] Make it possible to relabel target scrape query parameters.
+* [FEATURE] Add support for `A` records in DNS service discovery.
+* [ENHANCEMENT] Update vendored dependencies.
+* [ENHANCEMENT] Add compression to more HTTP handlers.
+* [ENHANCEMENT] Make -web.external-url flag help string more verbose.
+* [ENHANCEMENT] Improve metrics around remote storage queues.
+* [ENHANCEMENT] Use Go 1.5.1 instead of Go 1.4.2 in builds.
+* [ENHANCEMENT] Update the architecture diagram in the `README.md`.
+* [ENHANCEMENT] Time out sample appends in retrieval layer if the storage is
+  backlogging.
+* [ENHANCEMENT] Make `hash_mod` relabeling action use MD5 instead of FNV to
+  enable better hash distribution.
+* [ENHANCEMENT] Better tracking of targets between same service discovery
+  mechanisms in one scrape configuration.
+* [ENHANCEMENT] Handle parser and query evaluation runtime panics more
+  gracefully.
+* [ENHANCEMENT] Add IDs to H2 tags on status page to allow anchored linking.
+* [BUGFIX] Fix series file deletion behavior when purging archived series.
+* [BUGFIX] Fix scraper HTTP connection leak when encountering HTTP errors.
+* [BUGFIX] Fix error checking and logging around checkpointing.
+* [BUGFIX] Fix map initialization in target manager.
+* [BUGFIX] Fix draining of file watcher events in file-based service discovery.
+* [BUGFIX] Add `POST` handler for `/debug` endpoints to fix CPU profiling.
+* [BUGFIX] Fix several flaky tests.
+* [BUGFIX] Fix busylooping in case a scrape configuration has no target
+  providers defined.
+* [BUGFIX] Fix exit behavior of static target provider.
+* [BUGFIX] Fix configuration reloading loop upon shutdown.
+* [BUGFIX] Add missing check for nil expression in expression parser.
+* [BUGFIX] Fix error handling bug in test code.
+* [BUGFIX] Fix Consul port meta label.
+* [BUGFIX] Fix lexer bug that treated non-Latin Unicode digits as digits.
+* [CLEANUP] Update build environment scripts and Makefiles to work better with
+  native Go build mechanisms and new Go 1.5 experimental vendoring support.
+* [CLEANUP] Remove unused .pkgignore file.
+* [CLEANUP] Remove logged notice about 0.14.x configuration file format change.
+* [CLEANUP] Move scrape-time metric label modification into SampleAppenders.
+* [CLEANUP] Switch from `github.com/client_golang/model` to
+  `github.com/common/model` and related type cleanups.
+* [CLEANUP] Switch from `github.com/client_golang/extraction` to
+  `github.com/common/expfmt` and related type cleanups.
+* [CLEANUP] Exit Prometheus when the web server encounters a startup error.
+* [CLEANUP] Simplify Travis CI configuration.
```

Comment:
Stuff like that has nothing to do with the behavior of the Prometheus application. Personally, I wouldn't list such things here (similarly the .pkgignore thing) – the list is nosiy enough already.